service/controller/v2/arm_templates/main.json

```diff
@@ -362,12 +362,14 @@
           "vmssOsImageOffer": {"value": "[parameters('masterNodes')[0].osImage.offer]"},
           "vmssOsImageSKU": {"value": "[parameters('masterNodes')[0].osImage.sku]"},
           "vmssOsImageVersion": {"value": "[parameters('masterNodes')[0].osImage.version]"},
+          "vmssOverprovision": {"value": "false"},
```

Comment:
we don't need overprovision for masters as by default it creates two vms and both can be up at the same time, messing up load balancers.